tests/runtime-tests.sh

```diff
@@ -7,14 +7,15 @@ fi
 
 set -e;
 
-pushd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1
-
-BPFTRACE_RUNTIME_TEST_EXECUTABLE=${BPFTRACE_RUNTIME_TEST_EXECUTABLE:-@CMAKE_BINARY_DIR@/src/bpftrace};
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+BPFTRACE_RUNTIME_TEST_EXECUTABLE=${BPFTRACE_EXECUTABLE:-$DIR/../src/bpftrace};
 export BPFTRACE_RUNTIME_TEST_EXECUTABLE;
 
+pushd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1
+
 echo "===================="
 echo "bpftrace --info:"
 echo "===================="
-"${BPFTRACE_RUNTIME_TEST_EXECUTABLE}" --info
+$BPFTRACE_RUNTIME_TEST_EXECUTABLE --info;
```

Comment:
Why this change? You've technically made it less correct, if the directory containing the shell script happens to have any spaces :)

Same for `DIR` above. I would just split this up and avoid repeating:

```
TESTS_DIR="$(dirname "${BASH_SOURCE[0]}")";
BUILD_DIR="$(dirname "${TESTS_DIR}");
BPFTRACE_RUNTIME_TEST_EXECUTABLE="${BPFTRACE_EXECUTABLE:-$BUILD_DIR/src/bpftrace}";

"${BPFTRACE_RUNTIME_TEST_EXECUTABLE}" --info;
cd "${TESTS_DIR}" && python3 -u runtime/engine/main.py "$@"
```

Reply:
Ack. I was just copying what we had in the other scripts, which also should be fixed.